scripts/upgrade

```diff
@@ -94,17 +104,17 @@ ynh_add_nginx_config "path_no_root"
 #=================================================
 # CREATE DEDICATED USER
 #=================================================
-ynh_print_info "Making sure dedicated system user exists..."
+ynh_script_progression --message="Making sure dedicated system user exists..."
 
 # Create a dedicated user (if not existing)
-ynh_system_user_create $app
+ynh_system_user_create --username=$app
 
 #=================================================
 # SPECIFIC UPGRADE
 #=================================================
 # UPGRADE SEARX IN ITS VIRTUALENV
 #=================================================
-ynh_print_info "Upgrading Searx..."
+ynh_script_progression --message="Upgrading Searx..." --weight=7
 
 rm -r $final_path/lib/python2.7/site-packages/setuptools $final_path/lib/python2.7/site-packages/setuptools-*
```

Comment:
Why not `ynh_secure_remove` ?

Reply:
It won't probably work with `ynh_secure_remove`, because the helper will use double quotes, which doesn't work for wildcard.
I think it would work by using extglob though.

Reply:
Something like that:
```
ynh_secure_remove $final_path/lib/python2.7/site-packages/setuptools
for f in $(ls -d setuptools-*)
do
    ynh_secure_remove $f
done
```

Reply:
Is it really useful ?
Would probably be better to allow ynh_secure_remove to use wildcard, don't you think ?

Reply:
Sure!

Reply:
Have a look to `shopt -s extglob`